paket.dependencies

```diff
@@ -1,7 +1,8 @@
 source https://nuget.org/api/v2
```

Comment:
Using "frameworks: net46" in your paket.dependencies makes the resolutions simpler and downloads fewer packages